oap-server/analyzer/meter-analyzer/src/main/java/org/apache/skywalking/oap/meter/analyzer/prometheus/PrometheusMetricConverter.java

```diff
@@ -71,7 +71,7 @@ public String load(final String name) {
 
     public PrometheusMetricConverter(Rule rule, MeterSystem service) {
         this.convert = new MetricConvert(rule, service);
-        this.metricsNameEscapePattern = Pattern.compile("\\.");
+        this.metricsNameEscapePattern = Pattern.compile("[/.]");
```

Comment:
I am not good at regex, is this testes? Do we have UT for this?

Reply:
I have tried to write unit tests, but creating PrometheusMetricConverter object was complicated. So I have tested it locally.